tests/test_utils/reset_warning_registry.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import sys
+
+
+# We need to explicitly clear the warning registry context
+# https://docs.python.org/2/library/warnings.html
```

Comment:
Please keep within 90 columns

Reply:
Thanks, I've shortened the text. How about the url to Stackoverflow, is it ok to keep it on one line? Feels a bit awkward to break the url.

Reply:
Maybe add a # noqa ?

Reply:
I've checked. I reduced all the docstring to < 80 lines. Flake8 does not complain about the comments, so that should be ok.